### ISSUE TITLE
Pod update failed because podspec not found

### ISSUE THREAD

**Issue by beckyhilton** [capacitor-community/native-market#1]
**Describe the bug**
When I try to run either a pod update or a npx cap sync I get the following error:
```
[error] Error running update: Analyzing dependencies
Fetching podspec for `Capacitor` from `../../node_modules/@capacitor/ios`
Fetching podspec for `CapacitorCommunityNativeMarket` from `../../node_modules/@capacitor-community/native-market`
[!] No podspec found for `CapacitorCommunityNativeMarket` in `../../node_modules/@capacitor-community/native-market`
```

**To Reproduce**
1. In the console type 'npm install @capacitor-community/native-market'
2. then type 'npx cap sync'
3. See error
```
✖ Updating iOS native dependencies with "pod install" (may take several minutes): 
✖ update ios: 
[error] Error running update: Analyzing dependencies
Fetching podspec for `Capacitor` from `../../node_modules/@capacitor/ios`
Fetching podspec for `CapacitorCommunityNativeMarket` from `../../node_modules/@capacitor-community/native-market`
[!] No podspec found for `CapacitorCommunityNativeMarket` in `../../node_modules/@capacitor-community/native-market`

```
**Expected behavior**
the sync would finish successfullt

**Screenshots**

**Desktop (please complete the following information):**
 - OS: iOs
 - Version: 10.15.3

**Additional context**

**Comment by priyankpat** [comment]
I've just created a new release [0.1.3](https://github.com/capacitor-community/native-market/releases/tag/v0.1.3) with this fix. Please give it a try.

**Comment by psenechal** [comment]
I renamed the podspec file to get around that, but I’ll update to the new version as well. Thanks for the quick fix.

Quick question...for Android I’m using com.domain.appName for the appId value and the actual numeric App ID for IOS. Is that correct? Documentation only had the com.domain.appName example, but I assumed it was supposed to be the numeric App ID for iOS.

Only asking because my latest test deploy doesn’t work and I don’t know why. The Cordova plugin worked fine, but I’d rather use your Capacitor plugin.

**Comment by psenechal** [comment]
Got a bit better of an error message for you:

`
  [error] - ERROR Error: Uncaught (in promise): TypeError: NativeMarket.openStoreListing is not a function. (In 'NativeMarket.openStoreListing({
                appId: applicationId
            })', 'NativeMarket.openStoreListing' is undefined)
capacitor://localhost/settings-settings-module-es2015.js:76:42
generatorResume@[native code]
capacitor://localhost/vendor-es2015.js:166206:75
ZoneAwarePromise@capacitor://localhost/polyfills-es2015.js:4066:41
__awaiter@capacitor://localhost/vendor-es2015.js:166202:36
executeListenerWithErrorHandling@capacitor://localhost/vendor-es2015.js:58417:26
wrapListenerIn_markDirtyAndPreventDefault@capacitor://localhost/vendor-es2015.js:58459:54
capacitor://localhost/vendor-es2015.js:94574:50
onInvokeTask@capacitor://localhost/vendor-es2015.js:75756:43
runTask@capacitor://localhost/polyfills-es2015.js:3273:61
invokeTask@capacitor://localhost/polyfills-es2015.js:3586:45
invokeTask@capacitor://localhost/polyfills-es2015.js:4727:24
globalZoneAwareCallback@capacitor://localhost/polyfills-es2015.js:4753:31
`

**Comment by priyankpat** [comment]
I've placed fixed for it and is now available for [v0.1.4](https://github.com/capacitor-community/native-market/releases/tag/v0.1.4). 

You can simply pass in the Apple ID value as a string. e.g. `{ appId: '1234564789', }`

**Comment by priyankpat** [comment]
Closing this issue if there are no other problems. Feel free to open it if needed.
